tfhe/src/integer/ciphertext/mod.rs

```diff
@@ -100,6 +138,12 @@ pub trait IntegerCiphertext: Clone {
     }
 }
 
+pub trait RadixIntegerCiphertext: IntegerCiphertext + Sync + Send {
```

Comment:
does it need the Sync and Send bounds ?

Reply:
It mainly to avoid having to put that bounds in any _parallel function that is (or will be made generic of T: RadixIntegerCiphertext)

Reply:
are people expected to implement this trait ? Would the Sync and Send bounds be too restrictive for some implementers ?

Probably a nitpick but feels worth discussing a bit